### PR TITLE
Get rid of pydantic dependency

### PR DESCRIPTION
**Pull request by tamirOK**

### CODE REVIEW

File: arku/utils.py [hunk 1]
```diff
@@ -100,3 +100,25 @@ def args_to_string(args: Sequence[Any], kwargs: Dict[str, Any]) -> str:
             arguments += ', '
         arguments += ', '.join(f'{k}={v!r}' for k, v in sorted(kwargs.items()))
     return truncate(arguments)
+
+
+def import_string(dotted_path: str) -> Any:
+    """
```
Comment (martyanov) [inline arku/utils.py:106]: Please include the original source.

Comment (tamirOK) [reply arku/utils.py:106]: Done

File: tests/test_utils.py [hunk 2]
```diff
@@ -97,31 +97,19 @@ def test_typing():
...
+    assert arku.utils.import_string('os.path') == os.path
 
-        @validator('redis_settings', always=True, pre=True)
-        def parse_redis_settings(cls, v):
-            if isinstance(v, str):
-                return RedisSettings.from_dsn(v)
-            else:
-                return v
 
-    s1 = Settings(redis_settings='redis://foobar:123/4')
```
Comment (martyanov) [inline tests/test_utils.py:111]: Why the tests are gone?

Comment (tamirOK) [reply tests/test_utils.py:111]: Modified this test to pass without Pydantic

**Review by martyanov — APPROVED** [review]

**Review by martyanov — APPROVED** [review]

**Pull request merged**